pkg/ebpf/events_processor.go

```diff
@@ -235,7 +237,14 @@ func (t *Tracee) processEvent(event *external.Event) error {
 			return err
 		}
 	case SchedProcessExitEventID:
-		t.procInfo.DeleteElement(event.HostThreadID)
+		if t.config.Capture.NetPerProcess {
```

Comment:
You don't guard in all places with NetPerProcess, and in the future, we might have other reasons to have a process tree that are not related to net events. Let's remove this line, and guard in bpf code instead

Reply:
i only check for NetPerProcess to know if i should close the pcap. 
deleting from the process tree is done anyway for this event.
i'll guard in the bpf code, but i think this check for NetPerProcess should stay here.